src/BizHawk.Client.Common/Api/HttpCommunication.cs

```diff
@@ -30,10 +32,14 @@ public HttpCommunication(Func<byte[]> takeScreenshotCallback, string getURL, str
 
 		public string ExecGet(string url = null) => Get(url ?? GetUrl).Result;
 
-		public string ExecPost(string url = null, string payload = "") => Post(
-			url ?? PostUrl,
-			new FormUrlEncodedContent(new Dictionary<string, string> { ["payload"] = payload })
-		).Result;
+		public string ExecPost(string url = null, string payload = "")
+		{
+			_client.DefaultRequestHeaders.ExpectContinue = payload.Length > ExpectContinueThreshold;
```

Comment:
I think putting this here instead of in `Post` where you had it before means that the setting will affect subsequent GET requests (or POST requests which use `Post` directly). Fixed in https://github.com/TASEmulators/BizHawk/commit/88c54d11b6d888182cac96f54679db9a68a34e56.

Reply:
Well, ExpectContinue only applyies to POST, not to GET, it is only for when you're sending a payload.